src/main/java/org/xolstice/maven/plugin/protobuf/ProtocCompileMojo.java

```diff
@@ -47,13 +47,27 @@
             defaultValue = "${project.build.directory}/generated-sources/protobuf/java"
     )
     private File outputDirectory;
+    
+    
+    /**
+     * Additional comma-separated options to be passed to the Java generator.
+     * <b>Cannot</b> contain colon (<tt>:</tt>) symbols.
+     */
+    @Parameter(
+            required = false,
+            property = "javaOptions"
+    )
+    private String javaOptions;
```

Comment:
Please introduce similar changes into `ProtocTestCompileMojo` for completeness/symmetry and add another integration test to cover the new case.

Reply:
Done. I kept the same property name, not sure if we should have two different for test and main?

Reply:
No, it is fine. I have just checked the JS goals, and there is no difference in option name between main and test.